### ISSUE TITLE
Take picture

### ISSUE THREAD

**Issue by macarron** [Shoety/cordova-plugin-inappbrowserxwalk#8]
I can't take picture from the inappweb view. But that's work on crosswalk engine. I can work on it but tell me if you have any idea.

regards.

**Comment by macarron** [comment]
I have this warning from debugger 

```
Got an activity result, but no plugin was registered to receive it.
```

**Comment by ClaasBrueggemann** [comment]
are you talking about taking screenshots?

**Comment by macarron** [comment]
No taking picture from camera. The camera view open well and the I can take a picture. The camera view close with no error, but my input field still empty and I've this warning in debugger.

**Comment by ClaasBrueggemann** [comment]
So you are trying to take a camera picture in the inappbrowser?

**Comment by macarron** [comment]
Yes.
That's work in inappbrowser but not in inappbrowserxwalk...

I'm really sorry about my poor english...

**Comment by macarron** [comment]
This work in the XWalkWebView but not in the inappbrowserxwalk. I think it's a really little bug. But I can't find it

**Comment by ClaasBrueggemann** [comment]
So you are loading a website, which is using something like this:

input type="file" accept="image/*;capture=camera">
??

**Comment by macarron** [comment]
Yes my website have an input field that open a window to choose an action. Like taking photo from camera or from library. Then the user choose an option and take a picture. After that i've got my field filled with the result.

Something like that in example 5
http://plugins.krajee.com/file-basic-usage-demo

This work everywhere except in the plugin.

**Comment by macarron** [comment]
Ok I've got it!!!

I'm working on other improvement, I'll commit it soon.

Regards.

**Comment by phic901** [comment]
@macarron @ClaasBrueggemann  Hello, I am going through same problem as you've had. I've embedded 'inappbroswerxwalk' plugin into my project. But, the input file tag triggered only once. And that doesn't even return the pictures which I selected. I guess you've gone through this problem as well.
I would like to know how to use 'input file' tag on cordova.
Please let me know if you know anything.... I am desperate... :(

**Comment by rafaelcanovas** [comment]
@macarron @phic901 

Having the same issue. <input type="file"> triggers only the first time, then I cannot touch on it again on the lifetime of the webview. I have to close the app and open again so I can use it again.

Is this related to the inappbrowserxwalk or the xwalk webview?

**Comment by diaclavijo** [comment]
Hello, 

I am having the same issue: I think this might be related to this -> https://crosswalk-project.org/jira/browse/XWALK-6456 

They have specified the same bug description.

But in the comments the way to reproduce it, does not imply using an external URL, so they are rendering a page inside Cordova. In this case, it works, the problem appears when the file is meant to be uploaded using **inappbrowserxwalk** 

I tried changing the permissions as shown in the ticket: 

    <uses-permission android:name="android.permission.CAMERA" />
    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />

But no success, it is still failing.
